tests/integration_tests/tests/agentless_tests/test_events.py

```diff
@@ -143,6 +143,7 @@ def test_events_with_logs(self):
         else:
             self.fail("Expected logs to be found")
 
+    @pytest.mark.skipif(reason='causes the DB deadlock in snapshot')
```

Comment:
You seem to have set this as a skipif without setting a condition for it to be skipped?